src/app/questionnaire-item/store/init-actions.ts

```diff
@@ -0,0 +1,74 @@
+import * as R from 'ramda';
+import { Action, LinkIdPathSegment } from '../types';
+import { setAnswers } from './action';
+import { getExistingValueProperty } from './transform-initial-state';
+
+interface SimplifiedResponseItem<T> {
+  linkId: string;
+  answer?: [
+    {
+      item?: T[];
+      [value: string]: any;
+    }
+  ];
```

Comment:
are we sure that this is exactly one?

Reply:
No, I wrongly used a tuple instead of an array.